docs/src/api/public.md

```diff
@@ -131,12 +131,8 @@ Filter = t -> t ∉ [System]
 ## [Logging](@id logging)
 
 ```@autodocs
-Modules = [InfrastructureSystems]
-Pages   = ["logging.jl"]
-Order = [:type, :function]
-Filter = t -> t ∈ [InfrastructureSystems.configure_logging,
-                   InfrastructureSystems.open_file_logger,
-                   InfrastructureSystems.MultiLogger,
-                   InfrastructureSystems.LogEventTracker,
-                   ]
+Modules = [PowerSystems]
+Pages = ["utils/logging.jl"]
+Public = true
+Private = false
```

Comment:
Is this correct? It doesn't seem to build correctly on my local system.

Reply:
Yes, it should be ok. I will check in the CI if it is failing